src/simulator/cobot_model.h

```diff
@@ -21,13 +21,19 @@ class CobotModel : public robot_model::RobotModel {
   config_reader::ConfigReader config_reader_;
   ros::Publisher odom_publisher_;
 
-  // Receives drive callback messages and stores them
+  // Receives drive callback messages and stores Line2f
   void DriveCallback(const ut_multirobot_sim::CobotDriveMsg& msg);
+  // Initialize associated template lines (shape of robot)
+  void SetTemplateLines(const float r, const int num_segments);
+  void Transform();
 
  public:
   CobotModel() = delete;
   // Intialize a default object reading from a file
   CobotModel(const std::vector<std::string>& config_files, ros::NodeHandle* n);
+  CobotModel(const std::vector<std::string> &config_files, ros::NodeHandle *n,
+             std::string topic_prefix);
+
```

Comment:
Not sure if we want two constructors. We can use default topic prefix, etc. Not sure about this point, need others for clarfication.

Reply:
I agree with Yifeng, instead of overloading the constructor it would be better to just use a default empty string value for the second constructor.